geonode/api/resourcebase_api.py

```diff
@@ -454,6 +454,12 @@ def get_list(self, request, **kwargs):
             collection_name=self._meta.collection_name)
         to_be_serialized = paginator.page()
 
+        bundles = [
```

Comment:
@simod thanks for the background information. Were there any metrics captured for a performance baseline?